odoo_install.sh

```diff
@@ -253,3 +267,24 @@ echo "Start Odoo service: sudo service $OE_CONFIG start"
 echo "Stop Odoo service: sudo service $OE_CONFIG stop"
 echo "Restart Odoo service: sudo service $OE_CONFIG restart"
 echo "-----------------------------------------------------------"
+
+if [[ $INSTALL_NGINX == "True" ]]; then
+    echo -e "\n==== Installing NGINX ===="
+    sudo apt install nginx
+    sed -i "s/{DOMAIN}/$DOMAIN/g" nginx
+    sudo cp nginx /etc/nginx/sites-available/odoo 
+    sudo rm /etc/nginx/sites-enabled/default
+    sudo ln -s /etc/nginx/sites-available/odoo /etc/nginx/sites-enabled/odoo
+
+    echo -e "\n---- SSL certificate ----"
+    sudo apt install software-properties-common -y
+    sudo add-apt-repository universe -y
+    sudo add-apt-repository ppa:certbot/certbot -y
+    sudo apt update
+
+    sudo apt install certbot
+    sudo certbot certonly --standalone --preferred-challenges http -d $DOMAIN -n --agree-tos --email info@onestein.nl
```

Comment:
Can you make email info@onestein.nl configurable? For example in a variable?

Moreover: did you try if this line works without first restarting nginx service?